docs/GooglePublisher-batchsink.md

```diff
@@ -28,6 +28,9 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 
 **Topic**: Name of the Google Cloud Pub/Sub topic to publish to.
 
+**Format**: Format of the data to read. Supported formats are avro, blob, tsv, csv, delimited, json,
```

Comment:
@snehalakshmisha shouldn't the sink say "publish" or "write" instead of "read"?  If so, the tooltip for the pubsub sink is wrong too. It says "read"